clearwater-socket-factory/clearwater-socket-factory-sig.conf

```diff
@@ -0,0 +1,23 @@
+# clearwater-socket-factory-sig
+#
+# This process runs as root in the signaling namespace, and enables processes
+# running in other namespaces to obtain sockets in this namespace.
+
+description "Clearwater signaling socket factory server"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+script
+  WHITELIST_DIR=/etc/clearwater-socket-factory/signaling.d
+  NAMESPACE=signaling
+
+  . /usr/share/clearwater/bin/clearwater-socket-factory-common
+
+  # Sleep for 5 seconds, to avoid upstart respawning socket-factory too quickly
+  sleep 5
+
+  /usr/share/clearwater/bin/run-in-signaling-namespace /usr/share/clearwater/bin/clearwater_socket_factory $DAEMON_ARGS
```

Comment:
Similarly, not sure why you're not just calling clearwater-socket-factory-sig-service.